en/console-and-shells/server-shell.rst

```diff
@@ -20,7 +22,9 @@ terminal.
 Changing the Port and Document Root
 ===================================
 
-You can customize the port and document root using options::
+You can customize the port and document root using options
```

Comment:
Shouldn't these paragraphs end with a `.` or a `:`?

Reply:
I see, you're right.  I missed that the :: would generate a : at the end of the sentence.